src/main/java/com/byclosure/webcat/stepdefinitions/helpers/SeleniumInteractions.java

```diff
@@ -0,0 +1,192 @@
+package com.byclosure.webcat.stepdefinitions.helpers;
+
+import com.byclosure.webcat.context.IContext;
+import org.openqa.selenium.*;
+import org.openqa.selenium.internal.selenesedriver.TakeScreenshot;
+
+import java.util.List;
+import java.util.regex.Pattern;
+
+public class SeleniumInteractions {
```

Comment:
I see no point for "99%" of the methods on this class. We already have good, well stablished, interfaces provided by Webdriver. Why do we need to re-invent the wheel?